Simulation/Derivatives.m

```diff
@@ -17,8 +17,13 @@
   SimData.Plane.AoA = Pitch - flight_path_angle;
 
   [Drag, Lift] = AerodynamicModel(SimData);
-
-  F = Drag + Lift; % Net force on the object
+  
+  Thrust = ThrustModel(SimData);
+  
+  battery_update = Thrust(3);
+  
+  Thrust = [Thrust(1),Thrust(2)];
+  F = Drag + Lift + Thrust; % Net force on the object Commented out b/c Thrust causes imaginary numbers
```

Comment:
I think you meant to remove `Commented out b/c Thrust causes imaginary numbers
`